packages/button/src/Button.story.tsx

```diff
@@ -66,3 +73,14 @@ export const BaseGreen = Playground.bind({});
 BaseGreen.args = {
   variant: Variant.BaseGreen,
 };
+
+export const LoadingState = Playground.bind({});
+LoadingState.args = {
+  isLoading: true,
+};
+
+export const LoadingStateWithText = Playground.bind({});
+LoadingStateWithText.args = {
+  isLoading: true,
+  loadingText: 'Saving',
+};
```

Comment:
Just a note, I'm recommending (in the Chromatic adoption) moving away from stories where all that changes is the default args. We should make sure the knobs/controls are set up the way we want them, and allow users to toggle these states in the `Playground`/`LiveExample` story

Reply:
Gotcha - will keep that in mind for future PRs. I think the value of this approach is that it really highlights certain use cases, and in our current process of VD review, this is helpful for designers that aren't fully aware of how our props are set up to work. I'm sure this will be addressed largely by Chromatic itself, so I'm excited to see and discuss how our process would change.

Reply:
Would encourage us to adopt these patterns sooner rather than later so we don't need to make larger PRs down the line correcting them. If we know there's a convention that we're working towards, let's use it now.

Reply:
I agree from a process perspective, but it would just add overhead for designers without a Chromatic contingency. Since we're probably going to change a lot of things about our stories to accommodate Chromatic soon anyway, I'd push for an exception here. There are also other bugs I've identified following our migration to Storybook v7 that we'd have to make global bugfixes for anyway.

Reply:
I agree from a process perspective, but it would just add overhead for designers without a Chromatic contingency. Since we're probably going to change a lot of things about our stories to accommodate Chromatic soon anyway, I'd push for an exception here. There are also other bugs I've identified following our migration to Storybook v7 that we'd have to make global bugfixes for anyway.